README.md

```diff
@@ -40,10 +41,93 @@ logger.log(`Hi! I'm info log example`)
 logger.warn(`Hi! I'm warn log example`, { trace: false })
 logger.error(`Hi! I'm error log example`)
 logger.verbose(`Hi! I'm verbose log example`, { label: 'database' })
-logger.verbose(`Hi! I'm verbose log example #2`, { label: 'http' })
 logger.debug(`Hi! I'm debug log example`, { some: 'object' })
 ```
 
 ***output:***
 
-![](https://i.ibb.co/y63BtzS/image.png)
\ No newline at end of file
+![](https://i.ibb.co/y63BtzS/image.png)
+
+  
+## Environment Variables
+
+`NODE_ENV` production start does not show debug() logs
+
+`LOGARDIAN_LABELS` labels to be logged in the console. For example *LOGARDIAN_LABELS=database,http*
+
+  
+## FAQ
+
+#### How does it implement NestJS Logger without any framework libs?
+
+We made logger base on [LoggerService](https://github.com/nestjs/nest/blob/master/packages/common/services/logger.service.ts) but does not use it, so as not to include the NestJS libraries. But you can also use the Logardian instead of common NestJS logger.
```

Comment:
but we don't explicitly implement so that we stay dependless of NestJS libraries

Reply:
fixed